### PR TITLE
New mu-authorization configuration

### PR DESCRIPTION
**Pull request by Rahien**
We're keeping the old configuration around for use on the dev/test server for now as we don't want to expose our partners to the potential issues with a new mu-auth.

### CODE REVIEW

File: config/cl-authorization/config.lisp [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  :ext "http://mu.semte.ch/vocabularies/ext/"
+  :dct "http://purl.org/dc/terms/"
+  :adms "http://www.w3.org/ns/adms#"
+  :org "http://www.w3.org/ns/org#"
+  :person "http://www.w3.org/ns/person#"
+  :musession "http://mu.semte.ch/vocabularies/session/"
+  :foaf "http://xmlns.com/foaf/0.1/")
+
+(define-graph public ("http://mu.semte.ch/graphs/public")
+  ("ext:FileAddress" -> _)
```
Comment (ruben-bottu) [inline config/cl-authorization/config.lisp:52]: Not sure what the `->` syntax means

File: config/cl-authorization/config.lisp [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  ("http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#FileDataObject" -> _)
+  ("http://www.w3.org/ns/prov#Location" -> _)
+  ("besluit:Bestuurseenheid" -> _)
+  ("http://mu.semte.ch/vocabularies/ext/BestuurseenheidClassificatieCode" -> _)
```
Comment (ruben-bottu) [inline config/cl-authorization/config.lisp:56]: Might be usefull to use prefixes a much as possible to improve readability, e.g.
```
("ext:BestuurseenheidClassificatieCode" -> _)
```

**Comment by ruben-bottu** [comment]
Overall looks great! Some confusion about the `->` syntax though

File: config/cl-authorization/config.lisp (hunk 2 above)
Comment (Rahien) [reply config/cl-authorization/config.lisp:56]: true i will update this to use the prefixes

**Review by ruben-bottu — APPROVED** [review]
The application still seems to work as expected.

**Pull request merged**